mgr/cephadm: support for extra entrypoint args (#48801)
ceph/ceph: doc/cephadm/services/index.rst

   If there are fewer hosts selected by the placement specification than
   demanded by ``count``, cephadm will deploy only on the selected hosts.

.. _cephadm-extra-container-args:

Extra Container Arguments
=========================

.. warning:: 
  The arguments provided for extra container args are limited to whatever arguments are available for
  a `run` command from whichever container engine you are using. Providing any arguments the `run`
  command does not support (or invalid values for arguments) will cause the daemon to fail to start.

.. note::

>   For arguments for the process running inside the container rather than the for

Reviewer: ```suggestion
  For arguments passed to the process running inside the container rather than the for
```

      - "-v"
      - "/opt/ceph_cert/host.cert:/etc/grafana/certs/cert_file:ro"

.. _cephadm-extra-entrypoint-args:

Extra Entrypoint Arguments
=========================

.. note::

>   For arguments intended for the container runtime rather than the process inside
>   it, such as those required to limit the containers cpu or memory usage,
>   see :ref:`cephadm-extra-container-args`

Reviewer: ```suggestion
  For arguments passed to the container runtime rather than the process inside
  it see :ref:`cephadm-extra-container-args`
```

Similar to extra container args for the container runtime, Cephadm supports

> extra entrypoint args for appending arguments to the entrypoint process running
> within the container for a daemon cephadm deploys. For example, to set the
> collector textfile directory for node-exporter, one could apply a spec like

Reviewer: ```suggestion
appending to args passed to the entrypoint process running
within a container. For example, to set the
collector textfile directory for the node-exporter service, one could apply a service spec like
```
